common/src/main/java/tachyon/security/authorization/PermissionStatus.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.security.authorization;
+
+import java.io.IOException;
+
+import tachyon.Constants;
+import tachyon.conf.TachyonConf;
+import tachyon.exception.ExceptionMessage;
+import tachyon.security.LoginUser;
+import tachyon.security.authentication.AuthType;
+import tachyon.security.authentication.PlainSaslServer;
+
+public final class PermissionStatus {
+  private String mUserName;
+  private String mGroupName;
+  private FileSystemPermission mPermission;
+
+  /**
+   * Constructs an instance of {@link PermissionStatus}
+   * @param userName the user name
```

Comment:
About styling, in case you're using eclipse, you can use cmd+shift+F to auo-format the docs.